nodes/number/float_math.py

```diff
@@ -70,6 +81,8 @@ def edit(self):
             self.outputInteger = False
 
     def recreateOutputSocket(self):
+        self.labelOutputType = " (integer)" if self.outputInteger else " (float)" #just to be more clear
```

Comment:
maybe the 'labelOutputType' variable should just hold "Integer" or "Float". The brackets and spaces should be added in the draw_label function